src/services/filtering/FilterService.js

```diff
@@ -1,8 +1,6 @@
 import { EventEmitter } from 'eventemitter3';
 import FilterURLHandler from './FilterUrlHandler';
-import isEqual from 'lodash/isequal';
-import pickBy from 'lodash/pickby';
-import isEmpty from 'lodash/isempty';
+import { isEqual,pickBy,isEmpty } from 'lodash';
```

Comment:
can we get spaces in between multiple imports? I think that's how we do it elsewhere for consistency.

Reply:
Fixed!